test-self/all/methods.js

```diff
@@ -73,6 +74,7 @@ $x.slice();
 $x.stop();
 $x.toArray();
 $x.triggerHandler();
+$x.uniqueSort();
```

Comment:
This should be part of the 3.7 support change, shouldn't it?

Reply:
Not necessarily - we can be aware of jq3.7 without having a preset for it.